lib/recognizer/guardian.ex

```diff
@@ -28,12 +28,14 @@ defmodule Recognizer.Guardian do
   def encode_and_sign_access_token(access_token) do
     user = Keyword.get(access_token, :resource_owner)
 
+    roles = if Accounts.Role.admin?(user), do: ["staff"], else: []
```

Comment:
As we revisit roles in the future, we can revisit this and update it to support all of our roles. This will nice for apps consuming these JWTs, freeing them up from making API calls to check on roles.

Reply:
I'm happy with this. Our current roles are a mess, so only exposing the ones we plan on keeping here is :+1: